products/customer_analytics/frontend/components/ZendeskTicketsFilters/Priority.tsx

```diff
@@ -0,0 +1,34 @@
+import { useActions, useValues } from 'kea'
+
+import { LemonSelect } from '@posthog/lemon-ui'
+
+import { capitalizeFirstLetter } from 'lib/utils'
+
+import { zendeskTicketsFiltersLogic } from './zendeskTicketsFiltersLogic'
+
+export const PriorityFilter = (): JSX.Element => {
+    const { priority } = useValues(zendeskTicketsFiltersLogic)
+    const { setPriority } = useActions(zendeskTicketsFiltersLogic)
+
+    const options = ['all', 'low', 'normal', 'high', 'urgent']
+
+    const label = (key: string) => {
+        switch (key) {
+            case 'all':
+            case null:
+                return 'All priorities'
+            default:
+                return capitalizeFirstLetter(key)
+        }
+    }
```

Comment:
(similar to last comment) these could be outside the component to avoid re-creating the function every render